src/renderer/services/api/http/httpClient.ts

```diff
@@ -59,20 +59,22 @@ export class HttpClient {
         const start = Date.now();
         return {
             onComplete: ({request, status, config}: AxiosResponse): void => {
-                database.networkMetrics.addRecord(config.baseURL, {
-                    url: request.responseURL,
-                    code: status,
-                    latency: Date.now() - start,
-                    time: Date.now(),
-                });
+                if (process.env.NODE_ENV)
```

Comment:
== "production" ?

Reply:
this is intentional
we want this to be active in `production` and `development` but not when is used validator testing, when you start validator testing you use `ts-node` which does not provide this `NODE_ENV` variable

Reply:
It doesn't by default but it will send value if somebody has that env exported for some reason. I would be explicit here.